selfdrive/controls/controlsd.py

```diff
@@ -586,7 +586,8 @@ def state_control(self, CS):
     standstill = CS.vEgo <= max(self.CP.minSteerSpeed, MIN_LATERAL_CONTROL_SPEED) or CS.standstill
     CC.latActive = self.active and not CS.steerFaultTemporary and not CS.steerFaultPermanent and \
                    (not standstill or self.joystick_mode)
-    CC.longActive = self.enabled and not self.events.contains(ET.OVERRIDE_LONGITUDINAL) and self.CP.openpilotLongitudinalControl
+    CC.longActive = self.enabled and (not self.events.contains(ET.OVERRIDE_LONGITUDINAL) or self.CP.carName == 'toyota') \
+                   and self.CP.openpilotLongitudinalControl
```

Comment:
What is this for?

Reply:
reverted, tried to smooth gas presses out with this hack but doesn't seem to work all the time, and created other issues.